Doc/library/sqlite3.rst

```diff
@@ -2358,6 +2330,84 @@ can be found in the `SQLite URI documentation`_.
 .. _SQLite URI documentation: https://www.sqlite.org/uri.html
 
 
+.. _sqlite3-row-factory-how-to:
+
+How to work with row factories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, :mod:`!sqlite3` represent fetched rows as :class:`tuples <tuple>`.
+If a :class:`!tuple` does not suit your needs,
+use the built-in :class:`Row` type or a custom :attr:`~Connection.row_factory`.
+The former provides both indexed and case-insensitive named access to columns,
```

Comment:
```suggestion

:class:`!Row` provides both indexed and case-insensitive named access to columns,
```

IMO, this flows better and is cleaner as a separate paragraph, explicitly restating the name of the class in question rather than referring to it indirectly.

FYI, if you apply this suggestion directly, you'll need to add the line break manually, since GitHub strips white space around suggestions and I couldn't span it to the previous line as I would normally without conflicting with that suggestion.